docs/src/main/resources/microsite/includes/references.md

```diff
@@ -3,38 +3,38 @@
 [akka-http-json]: https://github.com/hseeberger/akka-http-json
 [akka-stream-json]: https://github.com/knutwalker/akka-stream-json
 [apache]: http://www.apache.org/licenses/LICENSE-2.0
-[api]: https://travisbrown.github.io/circe/api/#io.circe.package
+[api]: https://circe.github.io/circe/api/io/circe/package.html
 [archambault]: https://twitter.com/alxarchambault
 [argonaut]: http://argonaut.io/
 [argonaut-contributors]: https://github.com/argonaut-io/argonaut/graphs/contributors
 [argonaut-shapeless]: https://github.com/alexarchambault/argonaut-shapeless
 [argus]: https://github.com/aishfenton/Argus
-[benchmarks]: https://github.com/travisbrown/circe/blob/master/benchmark/src/main/scala/io/circe/benchmark/Benchmark.scala
+[benchmarks]: https://github.com/circe/circe/blob/master/benchmark/src/main/scala/io/circe/benchmark/Benchmark.scala
 [cats]: https://github.com/typelevel/cats
 [cats-xor]: http://typelevel.org/cats/tut/xor.html
-[circe-251]: https://github.com/travisbrown/circe/issues/251
-[circe-262]: https://github.com/travisbrown/circe/issues/262
-[circe-generic]: https://travisbrown.github.io/circe/api/#io.circe.generic.auto$
-[circe-jackson]: https://travisbrown.github.io/circe/api/#io.circe.jackson.package
-[circe-jawn]: https://travisbrown.github.io/circe/api/#io.circe.jawn.package
-[circe-parser]: https://travisbrown.github.io/circe/api/#io.circe.parser.package
+[circe-251]: https://github.com/circe/circe/issues/251
+[circe-262]: https://github.com/circe/circe/issues/262
+[circe-generic]: https://circe.github.io/circe/api/io/circe/generic/auto$.html
+[circe-jackson]: https://circe.github.io/circe/api/io/circe/jackson/index.html
+[circe-jawn]: https://circe.github.io/circe/api/io/circe/jawn/index.html
+[circe-parser]: https://circe.github.io/circe/api/io/circe/parser/index.html
```

Comment:
For some reason the circe-parser package doesn't show up in the scaladoc. Didn't investigate further.